theano/tensor/nnet/tests/test_nnet.py

```diff
@@ -1382,7 +1380,10 @@ def test_argmax_pushdown_bias():
     assert isinstance(fgraph.toposort()[1].op, tensor.CAReduce)
     assert isinstance(fgraph.toposort()[1].op.scalar_op, theano.scalar.Maximum)
     assert str(fgraph.toposort()[2].op) == 'OutputGuard'
-
+    assert check_stack_trace(
+        fgraph, ops_to_check=lambda node: (
+            isinstance(node, (SoftmaxWithBias, tensor.CAReduce)) or
+            isinstance(node.op.scalar_op, theano.scalar.Maximum)))
```

Comment:
I think that the second line is not needed, as it is already implied by the `isinstance(CAReduce)`. This was an additional check for node 1.